### PR TITLE
Add Article to JetNews

### PR DESCRIPTION
**Pull request by pedrosax**
Add Article to Jetnews and slight adjustment to colors applied to AppBar

### CODE REVIEW

**Review by pedrosax — COMMENTED** [review]
lgtm

**Review by pedrosax — COMMENTED** [review]
lgtm

**Review by pedrosax — COMMENTED** [review]
lgtm

File: JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt [hunk 1]
```diff
@@ -52,7 +52,7 @@ fun PostCardTop(post: Post, modifier: Modifier = Modifier) {
         val imageModifier = Modifier
             .heightIn(min = 180.dp)
             .fillMaxWidth()
-            .clip(shape = MaterialTheme.shapes.medium)
+            .clip(shape = MaterialTheme.shapes.large)
```
Comment (mlykotom) [inline JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt:55]: I think this should also apply to the `PostHeaderImage` in `PostContent.kt` ?

Comment (pedrosax) [reply JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt:55]: sure...fixed in updated commit

**Comment by pedrosax** [comment]
updated Android Studio article for JetNews

**Pull request merged**